build.sbt

```diff
@@ -68,6 +68,25 @@ lazy val cats = crossProject(JSPlatform, JVMPlatform)
 lazy val catsJVM = cats.jvm
 lazy val catsJS = cats.js
 
+lazy val cats2 = crossProject(JSPlatform, JVMPlatform)
+  .dependsOn(core)
+  .jvmSettings(Mima.settings)
+  .jsSettings(Mima.settings)
+  .settings(
+    shared,
+    name := "case-app-cats-effect-2",
+    Compile / scalaSource := file((catsJVM / Compile / scalaSource).value.toString.replace("jvm", "shared")),
```

Comment:
I'm not sure this is going to work. Something like `Compile / sources := (catsJVM / Compile / sources).value` should I think.

Reply:
Thanks, that's a lot nicer.